egret/model_library/unit_commitment/services.py

```diff
@@ -272,30 +272,25 @@ def _check_for_requirement( requirement ):
         raise Exception('Exception adding ancillary_services! ancillary_services requires one of: garver_3bin_vars, garver_2bin_vars, garver_3bin_relaxed_stop_vars, ALS_state_transition_vars, to be used for the status_vars.')
 
     ## set some penalties by default based on the other model penalties
-    default_reg_pen = value(model.LoadMismatchPenalty+model.ReserveShortfallPenalty)/2.
     ## set these penalties in relation to each other, from higher quality service to lower
     model.RegulationPenalty = Param(within=NonNegativeReals,
-            initialize=system.get('regulation_penalty_price', default_reg_pen),
+            rule=make_penalty_rule(model, 'regulation_penalty_price', 4.),
```

Comment:
General comment: these are sufficiently important constants that I think they warrant a big comment block in the corresponding files, defining the values and the hierarchy/presence.